client/my-sites/email/titan-setup-thank-you/index.tsx

```diff
@@ -104,11 +104,6 @@ const TitanSetupThankYou = ( props: TitanSetupThankYouProps ): JSX.Element => {
 			showSupportSection={ true }
 			thankYouImage={ thankYouImage }
 			thankYouTitle={ translate( 'Congratulations on your purchase!' ) }
```

Comment:
I think this should be `'Your email is now ready to use'` or `'Your email is now ready to use!'`. The copy from the updated designs is intended for normal checkout, while we are showing this screen for the case where a user sets up their mailbox. (This is a tricky one, as we need the old copy and the new design!)

Reply:
I've reverted it back to the original, which was: _Your email is now ready to use_